### PR TITLE
Updated requirements.txt

### PR DESCRIPTION
**Pull request by 007vedant**
- upgraded pyyaml, python-dotenv, markdown2 libraries to their latest stable version
- tested application against any kinds of code break
- added .db to gitignore

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -1,11 +1,11 @@
 Flask==1.1.2
-python-dotenv==0.14.0
+python-dotenv==0.19.2
```
Comment (jberkus) [inline requirements.txt:2]: If possible, I'd like to set these as continously updated.  So

`python-dotenv>=0.19.2`

Comment (007vedant) [reply requirements.txt]: @jberkus I've the set the packages to continuous updates and have also upgraded the rest of them. Please have a look at the latest commit.

**Comment by jberkus** [comment]
ok, lemme test all this, and then we'll go for it.

**Comment by jberkus** [comment]
Tested on the new test server, works.

Merging.

**Pull request merged**